Workbooks/CosmosDb Fleet/Overview.workbook

```diff
@@ -0,0 +1,2315 @@
+                    "CollectionName"
+                  ],
+                  "splitBySortOrder": -1,
+                  "splitByLimit": 50
+                },
+                {
+                  "namespace": "microsoft.documentdb/databaseaccounts",
+                  "metric": "microsoft.documentdb/databaseaccounts-Requests-DocumentCount",
+                  "aggregation": 4,
+                  "splitBy": [
+                    "DatabaseName",
+                    "CollectionName"
+                  ],
+                  "splitBySortOrder": -1,
+                  "splitByLimit": 50
+                },
+                {
+                  "namespace": "microsoft.documentdb/databaseaccounts",
+                  "metric": "microsoft.documentdb/databaseaccounts-Requests-DataUsage",
+                  "aggregation": 4,
+                  "splitBy": [
+                    "DatabaseName",
+                    "CollectionName"
+                  ],
+                  "splitBySortOrder": -1,
+                  "splitByLimit": 50
+                },
+                {
+                  "namespace": "microsoft.documentdb/databaseaccounts",
+                  "metric": "microsoft.documentdb/databaseaccounts-Requests-IndexUsage",
+                  "aggregation": 4,
+                  "splitBy": [
+                    "DatabaseName",
+                    "CollectionName"
+                  ],
+                  "splitBySortOrder": -1,
+                  "splitByLimit": 50
+                },
+                {
+                  "namespace": "microsoft.documentdb/databaseaccounts",
+                  "metric": "microsoft.documentdb/databaseaccounts-Requests-ProvisionedThroughput",
+                  "aggregation": 3,
+                  "splitBy": [
+                    "DatabaseName",
+                    "CollectionName"
+                  ],
+                  "splitBySortOrder": -1,
+                  "splitByLimit": 50
+                },
+                {
+                  "namespace": "microsoft.documentdb/databaseaccounts",
+                  "metric": "microsoft.documentdb/databaseaccounts-Requests-UpdateAccountKeys",
+                  "aggregation": 7
+                },
+                {
+                  "namespace": "microsoft.documentdb/databaseaccounts",
+                  "metric": "microsoft.documentdb/databaseaccounts-Requests-NormalizedRUConsumption",
+                  "aggregation": 3,
+                  "splitBy": [
+                    "DatabaseName",
+                    "CollectionName"
+                  ],
+                  "splitBySortOrder": -1,
+                  "splitByLimit": 50
+                }
+              ],
+              "gridFormatType": 2,
+              "resourceLimit": 10000,
+              "filters": [
+                {
+                  "id": "1",
+                  "key": "CollectionName",
+                  "operator": 1,
+                  "values": [
+                    "<empty>"
```

Comment:
Did you want this filter applied? Since you removed this filter from cosmos db insights